ci-operator/step-registry/baremetalds/assisted/operator/setup/baremetalds-assisted-operator-setup-commands.sh

```diff
@@ -119,14 +119,14 @@ metadata:
  name: agent
```

Comment:
Not super important but the AgentServiceConfig isn't namespaced so you can drop the namespace.

Reply:
ack